cmd/preview/create.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2020 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package preview
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/okteto/okteto/pkg/analytics"
+	"github.com/okteto/okteto/pkg/cmd/login"
+	"github.com/okteto/okteto/pkg/log"
+	"github.com/okteto/okteto/pkg/okteto"
+	"github.com/spf13/cobra"
+)
+
+// Create creates a preview environment
+func Create(ctx context.Context) *cobra.Command {
+	var previewType string
+	cmd := &cobra.Command{
+		Use:   "create <name>",
+		Short: "Creates a preview environment",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if err := validatePreviewType(previewType); err != nil {
```

Comment:
are we going to handle the scenario where the version of Okteto Enterprise doesn't have this feature available?